public/components/event_analytics/explorer/sidebar/sidebar.tsx

```diff
@@ -87,24 +91,41 @@ export const Sidebar = (props: ISidebarProps) => {
     });
   };
 
+  const checkWithStoredFields = () => {
+    if (
+      explorerFields.selectedFields.length === 0 &&
+      storedExplorerFields.selectedFields.length !== 0
+    ) {
+      return storedExplorerFields;
+    } else {
```

Comment:
else can be removed? only with return statement?

Reply:
removed else